lib/bigquery/dataset.js

```diff
@@ -56,6 +56,9 @@ function Dataset(bigQuery, id) {
 /**
  * Create a table given a tableId or configuration object.
  *
+ * See [Tables: insert](https://cloud.google.com/bigquery/docs/reference/v2/tables/insert)
```

Comment:
http://jscs.info/rule/maximumLineLength.html - they have a `urlComments` flag to ignore long urls within comments!